api/src/pcapi/core/offers/api.py

```diff
@@ -1783,6 +1782,42 @@ def check_can_move_event_offer(offer: models.Offer) -> list[offerers_models.Venu
     if count_reimbursed_bookings > 0:
         raise exceptions.OfferHasReimbursedBookings(count_reimbursed_bookings)
 
+    return get_venues_with_pricing_points_for_same_offerer(offer)
+
+
+def check_can_move_event_collective_offer(offer: educational_models.CollectiveOffer) -> list[offerers_models.Venue]:
+    if not offer.isEvent:
+        raise exceptions.OfferIsNotEvent()
+
+    count_past_stocks = (
+        educational_models.CollectiveStock.query.with_entities(educational_models.CollectiveStock.id)
+        .filter(
+            educational_models.CollectiveStock.collectiveOfferId == offer.id,
+            educational_models.CollectiveStock.startDatetime < datetime.datetime.utcnow(),
+        )
+        .count()
+    )
+    if count_past_stocks > 0:
+        raise exceptions.OfferEventInThePast(count_past_stocks)
+
+    count_reimbursed_bookings = (
+        educational_models.CollectiveBooking.query.with_entities(educational_models.CollectiveBooking.id)
+        .join(educational_models.CollectiveBooking.collectiveStock)
+        .filter(
+            educational_models.CollectiveStock.collectiveOfferId == offer.id,
+            educational_models.CollectiveBooking.isReimbursed,
+        )
+        .count()
+    )
+    if count_reimbursed_bookings > 0:
+        raise exceptions.OfferHasReimbursedBookings(count_reimbursed_bookings)
+
+    return get_venues_with_pricing_points_for_same_offerer(offer)
+
+
+def get_venues_with_pricing_points_for_same_offerer(
```

Comment:
```suggestion
def get_venues_with_same_pricing_point(
```
Ils sont toujours forcément du même offerer